src/js/player.js

```diff
@@ -524,7 +524,15 @@ class Player extends Component {
     this.reportUserActivity();
 
     this.one('play', this.listenForUserActivity_);
-    this.on('fullscreenchange', this.handleFullscreenChange_);
+
+    if (FullscreenApi.fullscreenchange) {
+      this.on(FullscreenApi.fullscreenchange, this.handleFullscreenChange_);
+
+      if (IE_VERSION) {
+        this.on(document, FullscreenApi.fullscreenchange, this.handleFullscreenChange_);
+      }
```

Comment:
I assume IE does not trigger the previous listener at all? If it did, this would seem to have potential to call `handleFullscreenChange_` twice.

Reply:
Yeah, as far as I can tell IE11 only triggers the document level event in my testing.

Reply:
Cool. And I checked in IE11 and it works the same as other browsers.